### ISSUE TITLE
syntax error, unexpected 'return' (T_RETURN)

### ISSUE THREAD

**Issue by tanero** [mavinoo/laravelBatch#87]
When I try to run Batch:update method,  error message shown like below;

Laravel Framework 8.83.9


 ```
ParseError                                                                                                            
                                                                                                                         
  syntax error, unexpected 'return' (T_RETURN)                                                                           
                                                                                                                         
  at C:\www\myapp\vendor\mavinoo\laravel-batch\src\Batch.php:275
    271▕     {                                                                                                           
    272▕         // no need for the old validation since we now use type hint that supports from php 7.0                 
    273▕         // but I kept this one                                                                                  
    274▕         if (count($columns) !== count(current($values)) {
  ➜ 275▕             return false;
    276▕         }
    277▕
    278▕         $query = [];
    279▕         $minChunck = 100;

```

**Comment by nguyenbinhit** [comment]
You just need to add ')' as follows I have bolded it up. This bug has been fixed, but for some reason it hasn't been updated yet:
```
if (count($columns) !== count(current($values))**)** {
    return false;
}
```

**Comment by mavinoo** [comment]
hi @tanero @nguyenbinhit 
fixed syntax error [issues 86](https://github.com/mavinoo/laravelBatch/issues/86)
reinstall package

**Comment by tanero** [comment]
> hi @tanero @nguyenbinhit fixed syntax error [issues 86](https://github.com/mavinoo/laravelBatch/issues/86) reinstall package


@nguyenbinhit  @mavinoo 

I have checked the git history, the issue had been fixed by mavinio at version 2.3.2

but code changed at version 2.3.2 by YuzuruS but missing parantheses.

https://github.com/mavinoo/laravelBatch/commit/2c814b5387f02bd8441f0aaf5b53fd2135efa756
